bridges/AnidexBridge.php

```diff
@@ -0,0 +1,205 @@
+<?php
+class AnidexBridge extends BridgeAbstract {
+
+	const MAINTAINER = 'ORelio';
+	const NAME = 'Anidex';
+	const URI = 'https://anidex.info/';
+	const CACHE_TIMEOUT = 3600; // 1h
```

Comment:
This is the default value, so it can be omitted.

Reply:
Indeed. Removing.